opennebula/resource_opennebula_virtual_network.go

```diff
@@ -175,71 +174,13 @@ func resourceOpennebulaVirtualNetwork() *schema.Resource {
 				ConflictsWith: []string{"reservation_vnet", "reservation_size"},
 			},
 			"ar": {
-				Type:          schema.TypeList,
+				Type:          schema.TypeSet,
```

Comment:
doing this means order between terraform and opennebula is not guaranteed. This means each time a new change will be performed tfstate won't have same order than one and it can trigger a diff.

Reply:
I'm not sure that we understand the same way here
[From the terraform documentation on TypeSet](https://www.terraform.io/docs/extend/schemas/schema-types.html#typeset): 
```
TypeSet implements set behavior and is used to represent an unordered collection of items, meaning that their ordering specified does not need to be consistent, and the ordering itself has no impact on the behavior of the resource.
```
For TypeList:
```
Used to represent an ordered collection of items, where the order the items are presented can impact the behavior of the resource being modeled.
```

It appears to me that's with the TypeList that some diffs would occur.
If we agree that AR order doesn't have any impact on OpenNebula side, TypeSet seems ok to me.

Reply:
We need to perform real test on minione to validate this. I need to be convinced by the real behavior.